lib/travis/api/v3/query.rb

```diff
@@ -35,7 +35,7 @@ def %<method_name>s!
     @@prefixed_params_accessor = <<-RUBY
       def %<prefix>s_params
         @%<prefix>s ||= begin
-          params = @params.select { |key, _| key.start_with?('%<prefix>s.'.freeze) }
+          params.select { |key, _| key.start_with?('%<prefix>s.'.freeze) }
```

Comment:
This supports both the old params ('maximum_number_of_builds' => 4) and the new namespacing of params ('settings.maximum_number_of_builds' => 4)